index.js

```diff
@@ -64,7 +64,7 @@ import EventEmitter from 'wolfy87-eventemitter'
       var anchor = actions[n];
       var port = anchor.getAttribute("data-port");
       var protocol = anchor.getAttribute("data-protocol") || 'http:';
-      var hostPrefix = anchor.getAttribute("data-host-prefix") + '-' || '';
+      var hostPrefix = anchor.getAttribute("data-host-prefix").concat('-') || '';
```

Comment:
This will fail if the attribute is not defined since `getAttribute` will return `null`. Javascrtip will error with `Cannot read property 'concat' of null`

Reply:
😭

Reply:
```suggestion
      var hostPrefix = anchor.getAttribute("data-host-prefix") && anchor.getAttribute("data-host-prefix").concat('-') || '';
```

Reply:
lol we end up with the same :)

Reply:
you were just pretty much faster :)